stdlib/_osx_support.pyi

```diff
@@ -15,7 +15,7 @@ def _find_executable(executable: str, path: str | None = None) -> str | None: ..
 def _read_output(commandstring: str, capture_stderr: bool = False) -> str | None: ...  # undocumented
 def _find_build_tool(toolname: str) -> str: ...  # undocumented
 
-_SYSTEM_VERSION: str | None  # undocumented
+_SYSTEM_VERSION: Final[str | None]  # undocumented
```

Comment:
I'd leave this one as-is. The value is modified and mutated in `_osx_support.py` at runtime, so I think it's debatable whether it's a constant that's set once and never modified: https://github.com/python/cpython/blob/8303d32ff55945c5b38eeeaf1b1811dbcf8aa9be/Lib/_osx_support.py#L84-L114

It's also an undocumented, private constant that we probably wouldn't add today if we were writing the stubs from scratch, so I think it's best if we just leave it alone unless the annotation here is causing an issue for somebody.

Reply:
https://github.com/python/typeshed/pull/12332/commits/f9deeb72a2182e991dbb698453ed5f88680c0edc